### DISCUSSION TITLE
Add a starlight + tailwind enabled version to your astro.new presets

### DISCUSSION THREAD

**Discussion by aschrijver** [withastro/starlight#1077] [Feature Requests]
### What version of `starlight` are you using?

3.5.3

### What is your idea?

Add a "Starlight + Tailwind" preset to https://astro.new/latest/getting-started to help with bug reporting.

### Why is this feature necessary?

In order to file a bug report a minimul reproduction must be provided. I tried doing so on Stackblitz and Gitpod, but both give different unrelated errors when following the [instructions](https://starlight.astro.build/guides/css-and-tailwind/#add-tailwind-to-an-existing-project) (first command `npx astro add tailwind` already fails). Friction to report a bug is just high without a preset environment.

Related support question: https://discord.com/channels/830184174198718474/1174038331340890252

### Participation

- [ ] I am willing to submit a pull request for this feature.

**Comment by HiDeoo** [comment]
Not to dismiss the idea but more for information,  this is already possible using the https://astro.new/starlight-tailwind?on=stackblitz URL.

**Reply by aschrijver** [reply]
Yes, thank you. @delucis helped me on Discord and pointed to https://stackblitz.com/github/withastro/starlight/tree/main/examples/tailwind .. I will mark your response as the answer. (Oh, I can't)

**Comment by HiDeoo** [comment]
Closing this feature request as the *Starlight + Tailwind* template can now be found on astro.new in the *Templates* category: https://astro.new/latest/templates

<img width="412" alt="image" src="https://github.com/withastro/starlight/assets/494699/25153ae2-18aa-4183-ac19-f60c938a6a4e">

Thanks for the feedback 🙌
